### DISCUSSION TITLE
help to build and design small organic  application for using c#.NET  any sugeestions

### DISCUSSION THREAD

**Discussion by mattevenkateswarlu** [community/community#57714] [Programming Help]
1.need help for building web applications using c#

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @mattevenkateswarlu ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
